activerecord/test/cases/adapters/postgresql/schema_test.rb

```diff
@@ -480,6 +480,7 @@ class DefaultsUsingMultipleSchemasAndDomainTest < ActiveRecord::PostgreSQLTestCa
     @connection.create_table "defaults" do |t|
       t.text "text_col", default: "some value"
       t.string "string_col", default: "some value"
+      t.decimal "decimal_col", default: "3.14159265358979323846"
```

Comment:
Before this PR, default values for decimal columns are dumped as floats, which means if someone runs `rake db:schema:load` it will load the schema by using a float for the decimal default. That leads to a test failure because of rounding:

```
Default of decimal column was not correctly parsed.
--- expected
+++ actual
@@ -1 +1 @@
-#<BigDecimal:32c5e18,'0.3141592653 5897932384 6E1',36(36)>
+#<BigDecimal:3231150,'0.3141592653 589793E1',27(27)>
```

I've rewritten the test to match the new schema behavior in this PR. Now the test passes.